libairspy/src/iqconverter_float.c

```diff
@@ -208,6 +238,8 @@ static _inline float process_fir_taps(const float *kernel, const float *queue, i
 	float sum = acc.m128_f32[0];
 #endif
 
+#elif defined(USE_NEON)
+	float sum = vaddvq_f32(acc);
```

Comment:
Compilation fails on RaspberryPI. This instruction is from A64 instruction set (aarch64), while RaspberryPI is 32bit.https://developer.arm.com/architectures/instruction-sets/intrinsics/#q=vaddvq_f32

Reply:
Good catch!
I've committed some tweaks, which should hopefully fix the issue. Unfortunately, do not currently have 32bit ARM platform handy, so could not verify whether there are other fixes needed.